packages/components/src/resizable-box/style.scss

```diff
@@ -9,6 +9,10 @@
 	.components-resizable-box__container.is-selected & {
 		display: block;
 	}
+	// Show the resize handle if set in props
```

Comment:
Can you combine those 2 rules which set `display:block` into one? They seem to have the same purpose.

Reply:
Sure thing!